lib/svg-preview-view.js

```diff
@@ -81,9 +81,9 @@ class SvgPreviewView extends View {
     this.controls.find('a').on('click', (e) => {
       this.changeBackground($(e.target).attr('value'))
     })
-    this.zoomOutButton.on('click', (e) => this.zoom(-0.1))
+    this.zoomOutButton.on('click', (e) => this.zoom(9 / 10 - 1))
     this.resetZoomButton.on('click', (e) => this.zoomReset())
-    this.zoomInButton.on('click', (e) => this.zoom(0.1))
+    this.zoomInButton.on('click', (e) => this.zoom(10 / 9 - 1))
```

Comment:
As describe above of you use the argument `0.1` and `-0.1` the scaling factor is *not* symmetric. Clicking each zoom button (in and out) once does not get you back to the original zoom factor.

With the updated argument for zoom-in (0.111... instead of 0.1) the two scaling factors are exact inverse operations. So after clicking both you are exactly back to where you started.